### ISSUE TITLE
Server type alias

### ISSUE THREAD

**Issue by barksten** [modocache/signatures#1]
Hi!
I'm new to go and I found your blog post about a high coverage small RESTful app fitting to play around with. But some things doesn't work for me.
You made the type alias
`type Server *martini.ClassicMartini`
and then in `NewServer` you create an instance and call some martini functions.
That doesn't compile for me.
If I look at the type definition over at [martini](https://github.com/go-martini/martini/commit/e426b4fa2fd60243fef1174435ebb9ea1ffa2f6c) I find that it have it's martini instance embedded.
If I change your code to

```
func NewServer(session *DatabaseSession) Server {
    // Create the server and set up middleware.
    m := Server(martini.Classic())
    m.Martini.Use(render.Renderer(render.Options{
        IndentJSON: true,
    }))
```

it compiles but the rest of the test code have similar problems.
AFAIK the martini code has not changed regarding ClassicMartini.

EDIT:
For a quick reproduction:

```
$ go get github.com/modocache/signatures
# github.com/modocache/signatures/signatures
modocache/signatures/signatures/server.go:24: m.Use undefined (type Server has no field or method Use)
modocache/signatures/signatures/server.go:27: m.Use undefined (type Server has no field or method Use)
```

**Comment by billinghamj** [comment]
Just replace all uses of `m.Use` and `m.Run` with `m.Martini.Use` and `m.Martini.Run` respectively.

**Comment by modocache** [comment]
Yeah, sorry--this was made with the latest version of Martini that was available at the time. Thanks for the pointers, @billinghamj!
